GitHubContents.ps1

```diff
@@ -0,0 +1,122 @@
+function Get-GitHubContent {
```

Comment:
Any reason you decided not to include the `Uri` ParameterSet here?  Is it because you thought that it might get too confused with the `Path` parameter?

Reply:
No reason, i just overlooked that functionality, i'll add it.